p2plending/frontend/src/app/terms/Terms.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+
+const config = require("../../../config/index");
+const title = config.title;
+
+class Terms extends Component {
+  goBack = () => {
+    // eslint-disable-next-line react/prop-types
+    this.props.history.goBack();
+  };
+
+  render() {
+    return (
+      <div id="container p-4 my-2 mx-auto">
+        <div className="text-center rounded p-4 text-white bg-info" >
+          <button
+            className="btn btn-outline-dark m-2 position-absolute " style={{ top: 70, left: 10 }}
+            onClick={() => {
+              this.goBack();
+            }}
+          >
+            <i className="fa fa-arrow-circle-left mr-1"></i>
+            {''} Back
+          </button>
+          <span role="img" aria-label="Globex emoji" style={{ fontSize: 50 }}>
+            🌎
+          </span>
+          <h2 className="m-0">Terms</h2>
+          <div className="mx-auto" style={{ maxWidth: "500px" }}>
+            <span>
+              {title} is a Peer to Peer Book Exchange! Share and explore the books within your local community.{" "}
+              <span role="img" aria-label="Globex emoji">
+                📖
+              </span>
+            </span>
+          </div>
+        </div>
+        <div className="container">
+          <div className="p-4 rounded mx-auto" >
+            <h2>Terms</h2>
+             <p>
+              By downloading or using the app, these terms will
+              automatically apply to you - you should make sure therefore
+              that you read them carefully before using the app. 
+            </p> 
+            <p>
+              Open San Diego Org is committed to ensuring that the app
```

Comment:
Replace "Open San Diego Org" with "Open San Diego"

Reply:
Also lets hyperlink this first reference to Open San Diego to our website (https://opensandiego.org)

Reply:
done